src/decorators/withResources/Resource.js

```diff
@@ -226,6 +227,34 @@ export default class Resource {
     return cloned
   }
 
+  replace(data) {
+    if (!isAllowed(this, HTTP_VERB_PUT)) {
+      throw createConfigError(
+        `You tried to call \`.replace(${JSON.stringify(
+          data
+        )})\` but this method is currently not allowed.`
+      )
+    }
+
+    // Clone the current Resource
+    const cloned = overrideThen(clone(this), (originalThen) => ({
+      save() {
```

Comment:
Nit: `replace()`?

Reply:
Well; I kept it save() because it made more sense in my opinion to use
entity().replace(...).save()
instead of
entity().replace(...).replace()

Reply:
Yeah, was wondering because they all had a different method. But save also works for me